src/status_im/contexts/wallet/db.cljs

```diff
@@ -6,4 +6,5 @@
    :selected-networks (set constants/default-network-names)})
 
 (def defaults
-  {:ui {:network-filter network-filter-defaults}})
+  {:ui {:network-filter network-filter-defaults
+        :tokens-loading? true}})
```

Comment:
I have made changes to this key in PR: https://github.com/status-im/status-mobile/pull/20160

Reply:
You mean you renamed it?

Reply:
@OmarBasem  have you considered modifying the subscription instead?
In the subscription you could say:
 if it's `nil` or `true`   -> loading
 if `false` -> loaded.
 
 ofc, you should carefully check that this solution works all the time and we don't have unexpected loading cases.

BTW, this is a common issue for `loding?` flags, sometimes I prefer instead a `ready?` flag.

Reply:
> if it's nil or true -> loading

@ulisesmac why should we modify the subscription and have and extra condition check? I think simply setting `:tokens-loading? true` is cleaner, as we want the wallet to be in loading state when it starts

Reply:
Setting a value in db as `true` has the problem that before dispatching that event, the key is `nil`, for loading states sometimes is not the best.

We don't need to modify the subscription, I was just asking if you already considered that approach.

Please, take the approach that solves the problem better 👍